python/tk_multi_workfiles/entity_models/extended_model.py

```diff
@@ -0,0 +1,280 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+from sgtk.platform.qt import QtGui, QtCore
+
+shotgun_model = sgtk.platform.import_framework("tk-framework-shotgunutils", "shotgun_model")
+ShotgunEntityModel = shotgun_model.ShotgunEntityModel
+
+
+class ShotgunExtendedEntityModel(ShotgunEntityModel):
+    """
+    A Shotgun Entity model with updatable filters and a couple of added methods.
```

Comment:
same here - would be great with more info. Thinking we move this into sg utils at some point, it's great if we do it properly now. Instead of 'a couple of added methods', it would be great with a list of what was added. When is this class useful/preferred over the std models we already have? And a quick code example would be fantastic.